components/02-elements/rating/RatingRate.vue

```diff
@@ -0,0 +1,257 @@
+<template>
+  <fieldset class="fieldset">
+    <legend id="rating-field-label">
+      {{ legend }}
+    </legend>
+    <div
+      class="rating rating--rate"
+      role="listbox"
+      aria-required="true"
+      aria-labelledby="rating-field-label"
+    >
+      <rating-rate-item
+        v-for="(item, idx) in ratingItems"
+        :id="item.id"
+        :key="item.id"
+        :aria-label="item.labelText"
+        :name="item.name"
+        :active="rating >= idx + 1"
+        :selected="rating === idx + 1"
+        @select="onSelect(idx + 1)"
+      />
+    </div>
+  </fieldset>
+</template>
+
+<script>
+  import RatingRateItem from './RatingRateItem'
+
+  export default {
+    components: {
+      RatingRateItem
+    },
+    model: {
+      prop: 'rating',
+      event: 'change'
+    },
+    props: {
+      rating: {
+        type: Number,
+        default: 0
+      },
+      ratingItems: {
+        type: Array,
+        required: true
+      },
+      legend: {
+        type: String,
+        required: true
+      }
+    },
+    methods: {
+      onSelect(selectedIndex) {
+        this.$emit('change', selectedIndex);
+      }
+    }
+  }
+</script>
+
+<style lang="scss">
+  $rating__size                   : 95px !default;
+  $rating__item-size              : $rating__size / 5 !default;
+  $rating__size--rate             : 250px !default;
+  $rating__margin--rate           : $spacer--medium !default;
+  $rating__item-size--rate        : $rating__size--rate / 5 !default;
+  $icon__active-color             : '#fab216' !default;
+  $icon__active-color--with-border: '#fab216' !default;
+  $icon__inactive-color           : '#c2c1df' !default;
+  $rating__error-margin-bottom    : $spacer !default;
+  $rating__error-color            : $red !default;
+
+  .rating {
```

Comment:
styles shouldn't be considered as something global, so rating element should work without styles from this module.
You can enable `scoped` styles to be sure it works this way.